lightningd/channel/channel.c

```diff
@@ -211,6 +225,9 @@ static struct io_plan *setup_peer_conn(struct io_conn *conn, struct peer *peer)
 
 static void peer_conn_broken(struct io_conn *conn, struct peer *peer)
 {
+	send_channel_update(peer, true);
```

Comment:
This is OK for now, but eventually we'll need to send this from the master daemon: peer could die for any other reason.  Also, when we do reconnection, we probably shouldn't announce disabling of route unless we take a few seconds?